packages/fluentui/react-northstar-prototypes/src/prototypes/MenuList/MenuGroup.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import { MenuGroupContextProvider } from './menuGroupContext';
+
+let count = 0;
+
+export const MenuGroup = (props: any) => {
```

Comment:
why props here has any? same goes for `MenuSectionHeader`

Reply:
prototype hack session :-)

You're right, i should fix types